### ISSUE TITLE
Add some more commands

### ISSUE THREAD

**Issue by RenderBr** [Average-Org/TSEconomy#11]
### Add some commands to allow more user interactivity

**Essentials:**
- Allow ability for players (non-admins) to see balances of other user (will be permissible)
- Delete, add, resetserver sub-commands for bankadmin

**Might be cool:**
- /baltop - lists top players of currencies, more of a "flex" than having any practical usecase
- /trade - command. allows player to sell item stack they are holding while having security they will receive payment
- /contract - allows a player to charge another based on a contract, $x amount per interval. the other player will have to accept
- /debt & /loans

**Comment by Maxthegreat99** [comment]
for baltop (i assume itd work like /lb from SEconomy) we can add flags to sort by currency or from the poorest to richest.

one thing i saw about /lb is that it was the only source of statistics admins got, i planned to fix it and actually add stuff like trade / bank / discord integration stuff in the Shop plugin but ig we can make it work in TSEconomy, we can even have a Shop / Discord module, but idk if that would go too far for an API, SEconomy was just a framework for other plugins to implement dunno if TSEconomy should follow the same ideaology

**Comment by Maxthegreat99** [comment]
Also the issue id see with Debt / Loans is what would be stopping players from duping money by taking loans via new accounts, transfering the cash to their main and just making the new account disapear forever.

**Comment by Maxthegreat99** [comment]
Furthermore, could you elaborate your explanation on trading? it will be selling the items to actuals players or the world itself?

**Comment by RenderBr** [comment]
> Also the issue id see with Debt / Loans is what would be stopping players from duping money by taking loans via new accounts, transfering the cash to their main and just making the new account disapear forever.

Yeah, not sure. I think it'd be really cool but probably very exploitable, definitely on the later end of things

> Furthermore, could you elaborate your explanation on trading? it will be selling the items to actuals players or the world itself?

Honestly, we could do both. I was just thinking amongst players but that might be a good idea too, kinda like how Essentials worked in Minecraft
